.mergify.yml

```diff
@@ -17,6 +17,9 @@ queue_rules:
   - name: default
     conditions:
       # Conditions to get out of the queue (= merged)
+      - label!=DNM
+      - -closed
+      - -draft
```

Comment:
add a condition to have approval from the maintainers and contributors to make it more clear?

Reply:
https://github.com/ceph/ceph-csi/pull/2763 address or club the changes.. ptal.